backend/rent.py

```
@@ -0,0 +1,158 @@
+from flask import Flask,render_template,redirect,request,session,url_for,flash,jsonify
+from flask_cors import CORS
+import mysql.connector
+import bcrypt
+
+app = Flask(__name__)
+app.secret_key = 'MY_SECRET_KEY'
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "secret_key"

(S105)